queries/staking/utils.ts

```diff
@@ -30,3 +31,13 @@ export function getStakingApy(totalStakedBalance: number, weekCounter: number) {
 	const yearlyRewards = (startWeeklySupply * (1 - supplyRate ** 52)) / (1 - supplyRate);
 	return totalStakedBalance > 0 ? (yearlyRewards * STAKING_REWARDS_RATIO) / totalStakedBalance : 0;
 }
+
+export function cobbDouglas(feesPaid: number, stakedBalance: number) {
+	return feesPaid ** 0.7 * stakedBalance ** 0.3;
+}
+
+export function getTradingRewards(weekCounter: number) {
+	const supplyRate = 1 - DECAY_RATE;
+	const initialWeeklySupply = INITIAL_WEEKLY_SUPPLY;
+	return initialWeeklySupply * supplyRate ** weekCounter * TRADING_REWARDS_RATIO;
```

Comment:
Should probably also do this math in BigNumber

Reply:
Sure. Just curious if there are any benefits to doing this math in BigNumber as most of them are constant numbers and `weekCounter `is a small integer.

Reply:
Initial weekly supply is a large decimal and straight up floating point arithmetic is prone to errors. It's only for display so not a huge issue but it's good practice to use big number / wei when working with decimal maths. Can read a bit more here and you might of noticed funky stuff when doing things with decimals in js number before. https://floating-point-gui.de/